components/buttongroup/overview.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Overview
+page_title: ButtonGroup for Blazor Overview
+description: Overview of the ButtonGroup for Blazor
+slug: buttongroup-overview
+tags: telerik,blazor,Toggle,button,group
+published: True
+position: 0
+---
+
+# ButtonGroup Overview
+
+This article provides information about the ButtonGroup component and its core features.
+
+The ButtonGroup component is a container for a buttons that can be toggle buttons, and lets you [select one or more]({%slug buttongroup-selection%}), and respond to the [selection and click events]({%slug buttongroup-events%}). The buttons inside fill up the container, match the styling according to the [chosen theme]({%slug general-information/themes%}) and provide the regular button features like images and icons and the other parameters and attributes.
+
+In this article:
+
+* [Basic and Toggle Buttons](#basic-and-toggle-buttons)
+* [Disabled State](#disabled-state)
+* [Hide Buttons](#hide-buttons)
+* [Styling](#styling)
+
+## Basic and Toggle Buttons
+
+To add a Telerik ButtonGroup to your Blazor app:
+
+1. Add the `TelerikButtonGroup` tag.
+1. Inside it, add `ButtonGroupToggleButton` or `ButtonGroupButton` tags that denote each button.
+    * The `ButtonGroupToggleButton` becomes primary when clicked and de-selects when another one is clicked. Read more in the [Selection]({%slug buttongroup-selection%}) article.
+    * The `ButtonGroupButton` does not change its visual state when clicked.
+1. Use the `OnClick` event of these buttons to handle the user actions.
+
+>caption TelerikButtonGroup with regular buttons and toggle buttons, and their respective OnClick handlers
+
+````CSHTML
+@* Each individual button lets you control its selected state, have a click handler and template, icons, text *@
+
+<TelerikButtonGroup>
+    <ButtonGroupButton OnClick="@FirstClick">First button</ButtonGroupButton>
+    <ButtonGroupToggleButton OnClick="@SecondClick">Second button</ButtonGroupToggleButton>
+</TelerikButtonGroup>
+
+@code{
+    async Task FirstClick()
+    {
+        Console.WriteLine("the first button was clicked.");
```

Comment:
```suggestion
        Console.WriteLine("the first button was clicked.");
```

Should we add more info that the difference between the two types of buttons is the selection capabilities? That ButtonGroupButton is not selectable.

Reply:
This is handled in the Selection article and in the SelectedChanged event section, so I don't think we should add it here too.